ci/ray_ci/tester_container.py

```diff
@@ -12,20 +12,29 @@ class TesterContainer(Container):
     A wrapper for running tests in ray ci docker container
     """
 
-    def __init__(self, docker_tag: str) -> None:
+    def __init__(
+        self,
+        docker_tag: str,
+        shard_count: int = 1,
+        shard_ids: Optional[List[int]] = None,
+    ) -> None:
         super().__init__(docker_tag)
+        self.shard_count = shard_count
+        self.shard_ids = shard_ids or [0]
```

Comment:
why this is `[0]` when `shard_ids` is an empty list?

I feel that it should just panic?

maybe change it to:


````
if shard_ids is None:
   self.shard_ids = [0]
else:
   assert(self.shard_ids)
   self.shard_ids = shard_ids
````

Reply:
actually let use an empty list as a default; it is ok for the list to be empty